### PR TITLE
Restructures exhibitors_grouped_by_name

### PR DESCRIPTION
**Pull request by kierangillen**
- Restructures `exhibitors_grouped_by_name` to have one array of `exhibitor` objects and adds correct `profile_id`

screen shot 2019-02-19 at 2 31 57 pm

### CODE REVIEW

File: src/schema/fair.ts [hunk 1]
```diff
@@ -291,12 +291,26 @@ const FairType = new GraphQLObjectType({
...
-            },
-            profile_ids: {
-              type: new GraphQLList(GraphQLString),
-              description: "Partner/Exhibitor default profile id",
+              description: "The exhibitor data.",
+              type: new GraphQLList(
+                new GraphQLObjectType({
+                  name: "FairExhibitor",
+                  fields: {
+                    exhibitor_name: {
```
Comment (l2succes) [inline src/schema/fair.ts]: From a best practice / convention perspective, these fields should be named `id` and `name`

File: _schema.graphql [hunk 2]
```diff
@@ -3354,15 +3354,23 @@ type FairCounts {
...
   # Letter exhibitors group belongs to
   letter: String
 
-  # Exhibitors sorted by name
-  exhibitors: [String]
-
-  # Partner/Exhibitor default profile id
-  profile_ids: [String]
+  # The exhibitor data.
+  exhibitors: [FairExhibitor]
```
Comment (ashfurrow) [inline _schema.graphql]: This is going to be a breaking schema change, which _should_ be okay as long as we aren't consuming this part of the schema yet. Make sure to check Volt, Reaction, and Force before merging.

File: src/schema/fair.ts (hunk 1 above)
Comment (ashfurrow) [reply src/schema/fair.ts]: Agreed.

File: _schema.graphql (hunk 2 above)
Comment (kierangillen) [reply _schema.graphql]: It's not being used anywhere else.

**Comment by joeyAghion** [comment]
Because this is a breaking change and Reaction embeds the old [out-of-date] schema, this is causing compatibility failures on Force PRs (like https://github.com/artsy/force/pull/3582 and https://github.com/artsy/force/pull/3578). If it's not already, mind updating Reaction to match this new schema?

**Pull request merged**